sdk/pubkey/src/lib.rs

```diff
@@ -639,8 +773,8 @@ impl Pubkey {
             crate::syscalls::sol_log_pubkey(self.as_ref() as *const _ as *const u8)
         };
 
-        #[cfg(not(target_os = "solana"))]
-        crate::program_stubs::sol_log(&self.to_string());
```

Comment:
Side note for me: this is fine for now, but we should restore these stubs at some point, probably through a new crate which just contains the stubs.